pkg/backend/display/progress_test.go

```diff
@@ -154,13 +154,8 @@ func TestProgressEvents(t *testing.T) {
 					t.Run("plain", func(t *testing.T) {
 						suffix := fmt.Sprintf(".interactive-%vx%v-plain", width, height)
 						opts := defaultOpts()
-						opts.Color = colors.Never
-						opts.RenderOnDirty = false
 						opts.ShowResourceChanges = true
-						opts.IsInteractive = true
-						opts.ShowResourceChanges = true
-						testProgressEvents(t, path, accept, suffix, opts, width, height, true)
```

Comment:
Why is it ok to replace these tests with just the simpleRenderer?

Reply:
These tests were added by me for this purpose in another recent PR.

Reply:
Ah ok 👍